resources/css/style.css

```diff
@@ -85,3 +85,40 @@
   border: 0;
   z-index: 9;
 }
+.sha256sum_hash {
+    display: flex;
```

Comment:
you have different types of indentation in your patch

Reply:
also alpha order the properties

Reply:
Ok @jayvdb , I will make these changes and commit them soon :-)

Reply:
Some other properties that I haven't added are not in alpha order, should I arrange them too ?

Reply:
I have committed changes with alpha ordering of all the properties and proper indentation. Please check if any more change is needed :-)